projects/elife.yaml

```diff
@@ -1216,7 +1216,7 @@ elife-libraries:
             ec2:
                 type: t3.medium # 2 core, 8 GB of memory
             ext:
-                size: 10 # GB
```

Comment:
Would go even larger, 50 or 100 GB as it's a single node being started on demand, and disk space is cheap with respect to CPU time.

Reply:
https://github.com/elifesciences/builder-base-formula/blob/a695d9970284f10cd45142fa976811596329f53f/elife/external-volume.sls#L45 promises to take care of the resizing of the filesystem?

Reply:
I’ve increased to 50. I will leave any other adjustment to @lsh-0

Reply:
> https://github.com/elifesciences/builder-base-formula/blob/a695d9970284f10cd45142fa976811596329f53f/elife/external-volume.sls#L45 promises to take care of the resizing of the filesystem?

volume has to be resized while unattached unfortunately. `resizefs` should handle the rest when it comes back up.

Reply:
> volume has to be resized while unattached unfortunately.

Actually I think this only applies to certain operations on the *root* volume. I had to go through a song and dance recently when switching instances from t2 to t3.